Update event info with privacy information (#80)
techworkersco/twc-site-nl: content/events/2025-04-23/index.en.md

## Session Times

Join one of the following sessions (the content is the same):

- **Wednesday, April 23rd**: 19:00 - 20:00
- **Friday, April 25th**: 13:00 - 14:00

> The session is open to all and will be hosted on Google Meet. You are encourage to join from an incognito window with an alias/nickname, and to leave your camera/audio off.

Reviewer: Suggested slight reformulation, to indicate that it's fine but not required for people to join in this way 
```suggestion
The session is open to all and will be hosted on Google Meet. You're welcome to join from an incognito window with a pseudonym, and to leave your camera and audio off.
```
